game-core/src/test/java/games/strategy/util/PropertyUtilTest.java

```diff
@@ -1,124 +0,0 @@
-package games.strategy.util;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
-import org.junit.jupiter.api.Test;
-
-import games.strategy.triplea.attachments.RulesAttachment;
-
-/**
- * PropertyUtil test sets / gets variables via reflection.
- *
- * <p>
- * The set in PropertyUtil done through a setter method, the get is done by reading the private variable value directly.
- * </p>
- *
- * <p>
- * To test we set up some sample test objects and do read/set property operations on them.
- * </p>
- */
-public class PropertyUtilTest {
```

Comment:
If some of these tests exercised the methods you moved to `MapPropertyWrapper`, I would recommend moving the appropriate tests to `MapPropertyWrapperTest`.  The logic is sufficiently complex that the tests are probably useful, even if they're only being used by map making tools.